aceknow/settings.py

```diff
@@ -25,7 +25,7 @@
 SECRET_KEY = SECRET
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Don't change this in the project. We only change it when the project is ready to launch